<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageFileSystemExport</class>
 <widget class="QWizardPage" name="WizardPageFileSystemExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="title">
   <string>File System</string>
  </property>
  <property name="subTitle">
   <string>Export workspace to the local file system.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>To Directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditDestDirectory"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxOverwriteData">
        <property name="text">
         <string>Overwrite existing files without waring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxFolderStructure">
        <property name="text">
         <string>Create folder structure</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonBrowse</sender>
   <signal>released()</signal>
   <receiver>WizardPageFileSystemExport</receiver>
   <slot>browseTargetDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseTargetDirectory()</slot>
 </slots>
</ui>
